dask/dataframe/core.py

```diff
@@ -3446,8 +3446,13 @@ def unique(self, split_every=None, split_out=1):
         )
 
     @derived_from(pd.Series)
-    def nunique(self, split_every=None):
-        return self.drop_duplicates(split_every=split_every).count()
+    def nunique(self, split_every=None, dropna=True):
+        uniqs = self.drop_duplicates(split_every=split_every)
+        if dropna:
+            # count mimics pandas behavior and excludes NA values
+            return uniqs.count()
+        else:
+            return uniqs.size
```

Comment:
not sure if `size` is the best option here?

Reply:
Seems reasonable to me